src/ert/_c_wrappers/enkf/row_scaling.py

```diff
@@ -25,7 +25,7 @@ def assign(self, target_size, func):
         >>> ens_config = ert.ensembleConfig()
         >>> config_node = ens_config["PORO"]
         >>> field_config = config.node.getFieldModelConfig()
-        >>> grid = ert.eclConfig().grid
+        >>> grid = ert.ensembleConfig().grid
```

Comment:
haha good on you for fixing this one - i still don't know / understand what it does ^^

Reply:
This is pseudo only, but it could be even more accurate by accessing the ensembleConfig via ens_config instead. Thanks for raising awareness here.

Reply:
oh OK i really just find that comment block puzzling / amusing, since I don't understand it, but good that my silliness actually led to improvement : )